.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -526,9 +534,13 @@ jobs:
         if: matrix.running_mode == 'aot' && matrix.test_option == '$WASI_TEST_OPTIONS'
         run: sudo apt-get update && sudo apt install -y jq
 
+      - name: build socket api tests
+        run: ./build.sh
+        working-directory: ./core/iwasm/libraries/lib-socket/test/
+
       - name: run tests
         timeout-minutes: 10
-        run: ./test_wamr.sh ${{ matrix.test_option }} -t ${{ matrix.running_mode }}
+        run: ./core/iwasm/libraries/lib-socket/test/build.sh && ./test_wamr.sh ${{ matrix.test_option }} -t ${{ matrix.running_mode }}
```

Comment:
the build command here seems duplicated, the `build socket api tests` step already does that